src/main/resources/config/application-prod.yml

```diff
@@ -117,3 +117,4 @@ application:
         password:
         branch-name: master
         max-wait-time-second: 30
+        depth: 1
```

Comment:
I guess by default we shouldn't set depth to 1 for prod env. As it produce config repull/reclone all time when some service try to resolve different some different commit id. What you think if set 10 or 100 by default?